eris.go

```diff
@@ -60,6 +60,9 @@ func wrap(err error, msg string) error {
 				msg:    e.msg,
 				stack:  stack,
 			}
+		} else {
+			// insert the frame into the stack
```

Comment:
This was also a bug. We were previously adding wrap frames to global stacks (so if you printed a global error without modification, it would have these random wrap layers on it that were coming from previous wrap calls).